README.md

```diff
@@ -62,79 +43,97 @@ example your `config/settings.html` or other configuration driven items
 - config/settings.html
 ```
 
-Download all the theme files
 
-```
-theme download
-```
+#### Here is a Breakdown of the Fields:
 
-Upload a theme file
+`api_key`
+The API key generated in your private app.
 
-```
-theme upload assets/layout.liquid
-```
+`password`
+The password generated in your private app.
 
-Remove a theme file
+`store`
+The address of your store (note there is no leading http:// or https://)
 
-```
-theme remove assets/layout.liquid
-```
+`theme_id`
+The theme id of the theme that should be responding to commands. If no theme id is specified, the currently active theme will be modified.
 
-Completely replace shop theme assets with the local assets
+`whitelist_files`
+Specifies which files and directories should be responding to changes. If nothing is specified the  `assets/, config/, layout/, snippets/, templates/ and locales/` directories will be modified according to commands entered.
 
-```
-theme replace
-```
+`ignore_files`
+Specifies which files should be explicitly ignored and not affected by any commands entered
 
-Watch the theme directory and upload any files as they change
 
-```
-theme watch
-```
+## Commands
 
-Open the store in the default browser
+`theme download`
+Downloads the theme files from your store to your computer. 
 
-```
-theme open
-```
+`theme upload path/to/file`
+Uploads a single file to your store. You can also upload directories by using the wildcard character. The command `theme upload assets/*` will upload all files in the assets directory.
 
-Bootstrap a new theme with [Timber](http://www.shopify.com/timber)
+`theme remove path/to/file`
+Remove a single file from your store. You can also remove directories by using the wildcard character, similar to the theme upload command. The command theme `remove assets/*` will remove all files in the assets directory.  
 
-```
-# use latest stable
-theme bootstrap api_key password shop_name theme_name
+`theme replace`
+This command replaces all store files with local files; it ensures the store theme files mirrors the files locally. This command can be destructive so it should be used with caution. If you are familiar with FTP or SFTP to upload files to a server, this is similar but not exactly the same. In addition to transferring files, it will delete any theme files in your store that are not present locally.   
```

Comment:
Place the warning first and in bold the separate it with a new paragraph.

**This command can be destructive so it should be used with caution**